### PR TITLE
Reject ahelp and IC issues have a short cooldown

### PR DESCRIPTION
**Pull request by harmonyn**
:cl: Fun Police
tweak: Reject Adminhelp and IC Issue buttons have a cooldown.
/:cl:

Why this will benefit the game: https://tgstation13.org/phpBB/viewtopic.php?f=23&t=9977&view=unread#unread

### CODE REVIEW

**Comment by lzimann** [comment]
Not necessary, we can just deal with admins in case they do it, plus there's nothing wrong with it not having a cooldown.

**Comment by bgobandit** [comment]
counterpoint: there's no reason why these buttons need to be pressed more than once

**Comment by sawrge** [comment]
make reject ahelp not give back ahelp verb please

**Comment by Iamgoofball** [comment]
counterpoint 2: we'll stop adding checks when you people stop abusing your tools and accounting for yourselves

because adminns dont punish eachother unless they rat eachother out

it's a good old boys club

**Comment by bgobandit** [comment]
goof I'm still an admin

File: code/modules/admin/topic.dm [hunk 1]
```diff
@@ -36,6 +43,7 @@
 
 		message_admins("[key_name_admin(usr)] marked [C.key]'s admin help as an IC issue.")
 		log_admin_private("[key_name(usr)] marked [C.key]'s admin help as an IC issue.")
+		spamcooldown = world.time + 300 // 30 seconds
```
Comment (lzimann) [inline code/modules/admin/topic.dm]: 30 seconds cooldown of rejecting any ahelp(and not just the same) is just awful.

Comment (bgobandit) [inline code/modules/admin/topic.dm]: is there a way to affect a variable in this datum when an ahelp comes in, I tried some things but nothing seemed to work

Comment (bgobandit) [inline code/modules/admin/topic.dm]: alternatively would it help to be 15 seconds

**Comment by Iamgoofball** [comment]
you're one of the few good ones

**Pull request merged**